json_rpc/src/response.rs

```diff
@@ -0,0 +1,108 @@
+use std::borrow::Cow;
+
+use serde::{
+    de::{DeserializeOwned, Deserializer},
+    Deserialize, Serialize,
+};
+use serde_json::Value;
+use tracing::error;
+
+use super::{Error, JSON_RPC_VERSION};
+
+/// A JSON-RPC response.
+#[derive(Clone, Eq, PartialEq, Serialize, Deserialize, Debug)]
+#[serde(deny_unknown_fields, untagged)]
+pub enum Response {
+    /// A successful RPC execution.
+    Success {
+        /// The JSON-RPC version field.
+        #[serde(deserialize_with = "deserialize_jsonrpc_field")]
+        jsonrpc: Cow<'static, str>,
```

Comment:
It's always going to be the same constant, can't we use `&'static str`?

Reply:
Same reason as for the `Error` type :)